src/node_options.cc

```diff
@@ -500,8 +500,12 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::trace_warnings,
             kAllowedInEnvironment);
   AddOption("--unhandled-rejections",
-            "define unhandled rejections behavior. Options are 'strict' (raise "
-            "an error), 'warn' (enforce warnings) or 'none' (silence warnings)",
+            "define unhandled rejections behavior. Options are 'strict' "
+            "(always raise an error), 'throw' (raise an error unless "
+            "'unhandledRejection' hook is set; this is the default), 'warn' "
+            "(log a warning), 'none' (silence warnings), "
+            "'warn-with-error-code' (log a warning and set exit code 1 unless "
+            "'unhandledRejection' hook is set)",
```

Comment:
nit: It's maybe more readable to have the default value indication not in the middle.
```suggestion
            "define unhandled rejections behavior. Options are 'strict' "
            "(always raise an error), 'throw' (raise an error unless "
            "'unhandledRejection' hook is set), 'warn' "
            "(log a warning), 'none' (silence warnings), "
            "'warn-with-error-code' (log a warning and set exit code 1 unless "
            "'unhandledRejection' hook is set). Default is 'unhandledRejection'.",
```